src/interpolate/Lerp_color.rei

```diff
@@ -0,0 +1,11 @@
+type rgba('a) = {
+  r: 'a,
+  g: 'a,
+  b: 'a,
+  a: 'a,
+};
```

Comment:
This structure in Reason is called a **record**. The easiest mental model is probably JS objects, but they actually behave a bit differently. Records are immutable by default and _require_ a predefined type. They're quite memory efficient and safe, and a good choice for something like colors. BuckleScript now supports compiling them to pure JS objects but for awhile they were actually stored as arrays.